repose-aggregator/tests/performance-tests/test_vars/filters/scripting.yml

```diff
@@ -1,3 +1,8 @@
+# TODO: Remove this file? Don't want it to get picked up by the automated performance tests, but
+# TODO: keeping this file gives us a sensibly named way to run generic scripting tests.
+#
+# TODO: YAML does not seem to support include or inherit. As such, there is more duplication than
+# TODO: necessary because each scripting language only changes the "script_lang" variable.
```

Comment:
Just pointing this out. Removing this file would allow us to remove the exception case in the determination logic. On the other hand, keeping the file makes some sort of sense if we will ever run the scripting filter test manually.  I lean toward just removing this file, and having to use the predefined language files even if we ever run a scripting test manually.

To the second point, I haven't been able to find a good solution. If anyone knows comes up with something, just let me know.